src/lib/fast_check.py

```diff
@@ -119,7 +119,8 @@ def obfuscate_dynatrace_access_key(dynatrace_access_key: str):
     if len(dynatrace_access_key) >= 7:
         is_new_token = dynatrace_access_key.startswith("dt0c01.") and len(dynatrace_access_key) == 96
         if is_new_token:
-            return dynatrace_access_key[:10] + '*' * (len(dynatrace_access_key) - 13) + dynatrace_access_key[-3:]
+            # characters between dots are the public part of the token
+            return dynatrace_access_key.split('.')[1]
```

Comment:
So here we skip the "dtc01." part, correct? Wouldn't it be better to keep it just to make it clearer what this is?

Reply:
Sorry about the after-merge comment but you speedran the merge 😅

Reply:
Haha my fault, I will wait till the next day next time